### DISCUSSION TITLE
Can't find where presets are installed

### DISCUSSION THREAD

**Discussion by trueschool** [AidaDSP/aida-x#33] [Q&A]
Hi, 
I was able to successfully install AIDA-X on my Apple Silicon Mac, but when I try to select a preset it opens a file window and I don't know where to navigate to. Lil' help please? 
Thanks

**Comment by spunktsch** [comment]
Hey @trueschool, 

did you download some models for AIDA-X? Does the default tw40_california_clean_deerinkstudios load?
You can find them on our website https://aida-x.cc/#models or in the repo https://github.com/AidaDSP/aidadsp-lv2/tree/main/models/deer%20ink%20studios
